src/Model/Entity/Panel.php

```diff
@@ -46,10 +46,37 @@ class Panel extends Entity
      */
     protected function _getContent(mixed $content): string
     {
+        if (is_string($content)) {
+            $contentInflated = @gzinflate($content);
```

Comment:
According to https://www.php.net/manual/en/zlib.installation.php zlib is not installed by default via PHP.

To not force all users to install the php zlib extension I'd simply wrap that in a 
```
if(function_exists('gzinflate'))
``` 
to make sure this is actually present.